robot_framework/eflyt.py

```diff
@@ -0,0 +1,263 @@
+"""This module contains all logic related to the Eflyt system."""
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import TimeoutException
+from OpenOrchestrator.orchestrator_connection.connection import OrchestratorConnection
+from OpenOrchestrator.database.queues import QueueStatus
+from itk_dev_shared_components.eflyt import eflyt_case, eflyt_search
+from itk_dev_shared_components.eflyt.eflyt_case import Case
+from robot_framework import config, letters
+
+
+def filter_cases(cases: list[Case], previous_case_numbers: list[str]) -> list[Case]:
+    """Filter cases from the case table.
+
+    Args:
+        cases: A list of cases to filter.
+
+    Returns:
+        A list of filtered case objects.
+    """
+    case_types_to_skip = (
+        "Børneflytning 1",
+        "Børneflytning 2",
+        "Børneflytning 3",
+        "Barn",
+        "Mindreårig",
+        "Sommerhus",
+        "Nordisk land"
+    )
+
+    filtered_cases = [
+        case for case in cases
+        if "Logivært" in case.case_types and case.case_number not in previous_case_numbers
```

Comment:
This breaks the check_queue function because you skip over any and all cases that has been touched before